protx-client/src/components/ProTx/components/maps/MapProviders.jsx

```diff
@@ -4,104 +4,54 @@ export default function Maptiles() {
   const providers = [];
   const layers = {};
 
-  // Provider: Stamen.
-  // No key required.
   // eslint-disable-next-line no-unused-vars
-  const basemapToner = L.tileLayer(
-    'https://stamen-tiles-{s}.a.ssl.fastly.net/toner/{z}/{x}/{y}{r}.{ext}',
+  const tiles_CartoDBPositron = L.tileLayer('https://abcd.basemaps.cartocdn.com/light_all/{z}/{x}/{y}{r}.png',
```

Comment:
I am not 100% on the url for these tiles. The config block provided is this:

```
var CartoDB_PositronNoLabels = L.tileLayer('https://{s}.basemaps.cartocdn.com/light_nolabels/{z}/{x}/{y}{r}.png', {
	attribution: '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors &copy; <a href="https://carto.com/attributions">CARTO</a>',
	subdomains: 'abcd',
	maxZoom: 20
});
```

Since we don't do any subdomain interpolation, I put the `abcd` subdomain in the URL.
I don't know if that will work correctly.

Reply:
works 👍